src/admin-views/aggregator/tabs/import-form.php

```diff
@@ -69,7 +69,11 @@ class="tribe-ea-field tribe-ea-dropdown tribe-ea-size-large"
 							$disabled = ( isset( $option->disabled ) ? $option->disabled : null );
 							if ( ! $upsell && $disabled ) {
 								$upsell = true;
-								?><option value="redirect"><?php esc_html_e( 'Buy Event Aggregator for more event sources and automatic imports!', 'the-events-calendar' ); ?></option><?php
+								?><option value="redirect"><?php
+									esc_html_e( 'Buy Event Aggregator', 'the-events-calendar' );
```

Comment:
Asking this out of curiosity really, but why two translatable strings broken up by a pipe separator vs a single translatable string that includes the pipe separator?